### PR TITLE
Add N/A value to SHINE Significance

### PR DESCRIPTION
**Pull request by samuelscandrett**
#1229 

Add "N/A" value to SHINE Significance domain values.

image

### CODE REVIEW

**Comment by samuelscandrett** [comment]
@aj-he Merge conflicts sorted and N/A moved below "Low" in options order, so new order goes "N/A", "Low", "Medium", "High".

File: arches_her/pkg/graphs/resource_models/Artefact.json [hunk 1]
```diff
@@ -21711,6 +21718,11 @@
                     "alias": "shine_form",
                     "config": {
                         "options": [
+                            {
+                                "id": "e8d04473-da05-4cda-95f4-ebfd0ff28122",
+                                "selected": false,
+                                "text": "N/A"
+                            },
```
Comment (aj-he) [inline arches_her/pkg/graphs/resource_models/Artefact.json]: @SDScandrettKint - the addition N/A to the "Shine form" node is unexpected as it is a concept and the collection does not have any of the other High-Low items in. The PR is for a change to domain options only.

Can you double check this and the other models?

Comment (samuelscandrett) [reply arches_her/pkg/graphs/resource_models/Artefact.json]: Removed all options from shine_form leaving just the rdm reference in [34068a3](https://github.com/archesproject/arches-her/pull/1230/commits/34068a3815902b90a568cf3da089fc7bd40bb5d7).

Also ensured that N/A is always before Low in the order.
(Note that the order of options is "High" -> "N/A" in all files apart from in branches/Location Data where the order is the reverse "N/A" -> "High")

File: arches_her/pkg/graphs/resource_models/Activity.json [hunk 2]
```diff
@@ -14538,6 +14545,11 @@
...
                     "config": {
                         "options": [
+                            {
+                                "id": "e8d04473-da05-4cda-95f4-ebfd0ff28122",
+                                "selected": false,
+                                "text": "N/A"
+                            },
                             {
                                 "id": "3620365b-6294-4356-93c0-52f53a2e4a49",
                                 "selected": false,
```
Comment (aj-he) [inline arches_her/pkg/graphs/resource_models/Activity.json]: @SDScandrettKint this still seems to be a concept node with domain node config

Comment (samuelscandrett) [reply arches_her/pkg/graphs/resource_models/Activity.json]: Apologies, completely missed the Activity model.

**Review by aj-he — APPROVED** [review]
LGTM

**Pull request merged**